### PR TITLE
[AC-1660] Drop react-code-block & add new CodeBlock component

### PR DESCRIPTION
**Pull request by AhyoungRyu**
In terms of reducing redundant pkg dependency, dropping react-code-block library but adding a new CodeBlock component  instead. 

record
^ Here's how it works.

### CODE REVIEW

File: src/components/CodeBlock.tsx [hunk 1]
```diff
@@ -1,16 +1,82 @@
...
-      showLineNumbers={true}
-      codeBlock
-    />
+    <CodeContainer>
+      <CopyButton
+        onClick={() => {
+          handleCopy();
+        }}
+      >
+        {isCopied ? '✅' : '📋'}
```
Comment (bang9) [inline src/components/CodeBlock.tsx:77]: 🤣ㅋㅋㅋ 심플하고 좋네요 이모지 짱👍👍👍

File: src/components/CodeBlock.tsx [hunk 2]
```diff
@@ -1,16 +1,82 @@
...
+  height: 26px;
+  width: 26px;
+  padding: 2px;
+`;
+
+export const CodeBlock = ({ token }: { token: Token }) => {
+  const [isCopied, setIsCopied] = useState(false);
+  const handleCopy = async () => {
+    try {
+      await navigator.clipboard.writeText(code);
```
Comment (bang9) [inline src/components/CodeBlock.tsx:54]: `clipboard API` ios webkit 이 살~짝 아슬해보이네용
image

**Review by bang9 — APPROVED** [review]
lgtm!

File: src/components/CodeBlock.tsx (hunk 2 above)
Comment (AhyoungRyu) [reply src/components/CodeBlock.tsx:54]: https://developer.mozilla.org/en-US/docs/Web/API/Navigator/clipboard#browser_compatibility
맞아요 그부분이 조금 걸리긴 하지만.... 부가적인 기능이니 안고가려합니다 😂

**Pull request merged**